src/libraries/System.Threading.Tasks.Parallel/tests/System.Threading.Tasks.Parallel.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <StartupObject />
+    <_WasmPThreadPoolSize Condition="'$(MonoWasmBuildVariant)' == 'multithread'">32</_WasmPThreadPoolSize>
```

Comment:
A value this high may cause reliability issues

Reply:
this is temporary and not production code

Reply:
If we have deputy design, we could let emscripten to allocate pthreads on the fly. 
That said these unit tests are nasty about creating many threads.